src/app/shared/components/color-picker/color-picker.component.scss

```diff
@@ -0,0 +1,64 @@
+:host {
+  &.ng-invalid {
+    &.ng-touched {
+      input {
+        border-color: var(--danger-color);
+      }
+    }
+  }
+
+  &.ng-valid,
+  &.ng-untouched {
+    sofie-error-message {
+      display: none;
+    }
+  }
+}
+
+.color-picker {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  padding: 0.3rem 0.3rem;
+  font-weight: lighter;
+  font-size: 0.85rem;
+  font-family: Roboto, Arial, sans-serif;
+  border-radius: 4px;
+  cursor: pointer;
+  border: 1px solid var(--gray-color);
```

Comment:
It would be nice if related properties were grouped (border-radius and border).

Reply:
Updated